src/app/components/dashboard-apps-dock/no-apps-notification.html

```diff
@@ -1,4 +1,4 @@
-<div class="close top-right-corner" type="button" aria-label="Close" ng-click='toggle()'>x</div>
+<div class="close top-right-corner" type="button" aria-label="Close" ng-click='$parent.isPopoverShown = false'>x</div>
```

Comment:
I think that should work, but @alexnoox would ask you to put that in a method :) Can you do this quick change plz?

Reply:
Hi @cesar-tonnoir , thans for your comment. Only one quesiton.
 I would be creating a controller only to handle that, Is really worth it to add "so much code" only for this?

Reply:
It would be better to have a method `$parent.closePopover()`

Reply:
That's a better one! commit pushed @alexnoox